ShitpostBot/ShitpostBot.Worker/Internal/BotCommandHandlers/RepostMatchAllBotCommandHandler.cs

```diff
@@ -57,16 +57,20 @@ public RepostMatchAllBotCommandHandler(IPostsReader postsReader, IChatClient cha
                 return true;
             }
 
+            await chatClient.SendMessage(messageDestination, "Starting to match. This might take a while...");
```

Comment:
await chatClient.SendMessage(messageDestination, "Starting to match. Čekej píčo :PauseChamp: ...");